apps/test-app/frontend/vite.config.mts

```diff
@@ -6,27 +6,17 @@ import { defineConfig } from "vite";
 import { viteStaticCopy } from "vite-plugin-static-copy";
 import react from "@vitejs/plugin-react";
 
-const localeDirs = [
-  "./node_modules/@itwin/core-react/lib/public/locales",
-  "./node_modules/@itwin/core-frontend/lib/public/locales",
-  "./node_modules/@itwin/components-react/lib/public/locales",
-  "./node_modules/@itwin/presentation-components/lib/public/locales",
-];
-
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [
     react(),
     viteStaticCopy({
       targets: [
-        ...localeDirs.map((dir) => ({
-          src: dir,
+        {
+          // copy assets from `@itwin` dependencies
+          src: "./node_modules/@itwin/**/**/public/*",
```

Comment:
shouldn't this be `./node_modules/@itwin/**/public/**/*`?

Reply:
Changed `/**/**/` -> `/**/`. I guess I duplicated it accidentally while testing.

There should not be `public/**/*` at the end because it would copy public assets from subfolders into destination root e.g files from `public/locale/en/` would be copied to the root. `public/*`  allows to keep folder hierarchy.